games-arcade/ccatch/ccatch-5.0.recipe

```diff
@@ -0,0 +1,43 @@
+SUMMARY="Help a cat catch birds"
+DESCRIPTION="cCATch is a portation of the game CATch, written in c using SDL"
+HOMEPAGE="http://www.victorswelt.de/catch.html"
+COPYRIGHT="2020 Victor Gamper"
+LICENSE="GNU GPL v3"
+REVISION="1"
+srcGitRev="018c0fd7184e02992488a95ce514c3f1d77d7604"
+SOURCE_URI="https://github.com/TheZeldakatze/c-CATch/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="c5f0d7b696d641b6168b92b05e822f76c92de31c784739d94f901b5596e593c5"
+SOURCE_FILENAME="c-CATch-$srcGitRev.tar.gz"
+SOURCE_DIR="c-CATch-$srcGitRev"
+# 6 Dec 2021 commit.
+ARCHITECTURES="all"
+
+PROVIDES="
+	cCATch = $portVersion
+	app:cCATch = $portVersion
+	"
+REQUIRES="
+	haiku
+	lib:libsdl$secondaryArchSuffix
+        "
+
+BUILD_REQUIRES="
+	haiku_devel
+	devel:libsdl$secondaryArchSuffix
```

Comment:
This should be `libSDL_1.2` (in both cases) as mentioned in: https://github.com/haikuports/haikuports/issues/3878